ui/client/actions/nk/nodeDetails.ts

```diff
@@ -44,19 +45,21 @@ export function nodeValidationDataClear(nodeId: string): NodeValidationClear {
 //TODO: use sth better, how long should be timeout?
 const validate = debounce(async (processId: string, validationRequestData: ValidationRequest, callback: (data: ValidationData, nodeId: NodeId) => void) => {
   const nodeId = validationRequestData.nodeData.id
-  const {data} = await HttpService.validateNode(processId, validationRequestData)
-  callback(data, nodeId)
+  const nodeType = NodeUtils.nodeType(validationRequestData.nodeData);
+  if (nodeType === "Properties") {
```

Comment:
Do we have constant/method for this check?

Reply:
Yes, we do. 
Done.